crates/uv-python/src/discovery.rs

```diff
@@ -1096,6 +1096,19 @@ impl PythonRequest {
         if value_as_path.is_file() {
             return Self::File(value_as_path);
         }
+
+        // e.g. path/to/python on Windows, where path/to/python is the true path
+        #[cfg(windows)]
+        if !value_as_path
+            .extension()
+            .is_some_and(|ext| ext == EXE_SUFFIX)
```

Comment:
We'll take this branch if we have "python.foo" with this logic, right? (which seems incorrect)

Reply:
(Changed to `is_none`.)

Reply:
Great thanks!